.gitattributes

```diff
@@ -0,0 +1,2 @@
+# Always keep main's version of versions.json during merges
+version.json merge=ours
```

Comment:
This might lead to weirdness if people merge main into their feature branch, although it certainly makes merging the release branch into main easier.

Reply:
Or maybe that wouldn't count as a "merge" since it's a fast-forward?